src/components/structures/TimelinePanel.tsx

```diff
@@ -1550,8 +1550,8 @@ class TimelinePanel extends React.Component<IProps, IState> {
         const isNodeInView = (node) => {
             if (node) {
                 const boundingRect = node.getBoundingClientRect();
-                if ((allowPartial && boundingRect.top < wrapperRect.bottom) ||
-                    (!allowPartial && boundingRect.bottom < wrapperRect.bottom)) {
+                if ((allowPartial && boundingRect.top <= wrapperRect.bottom) ||
+                    (!allowPartial && boundingRect.bottom <= wrapperRect.bottom)) {
```

Comment:
hmm, I didn't notice this condition in the last review - it's confusing style-wise. Can we make it look like the [style guide](https://github.com/vector-im/element-meta/pull/216)'s approach?

```
if (
  condition
  || condition
) {
  return true;
}
```

I'm less opinionated about the `||` being on the line before or after - would like it broken out either by lines or into variables though.

Reply:
I didn't know we mandated this, so I tend to leave existing code the way it was before but I definitely agree - this is much better!